### DISCUSSION TITLE
picker相关组件，展示不了其他选项

### DISCUSSION THREAD

**Discussion by miaoyaoxuan** [ant-design/ant-design-mobile#6457] [Q&A]
官网上代码复制出来的，但是效果不一样
![PS1C1@ 3}CZ5 T6V%SL1ZP1](https://github.com/ant-design/ant-design-mobile/assets/49223738/d3b45394-166e-4642-885d-7f15e0a0e9e0)
![C2PT4H_~NW`25NO(6LX~@0A](https://github.com/ant-design/ant-design-mobile/assets/49223738/bcfbf63e-2462-46c9-9bf7-e082c4adcbec)

**Comment by Tea-9-xiaoyu** [comment]
我也碰到这个问题了 兄弟怎么解决的
